src/main/views/subscription-management.njk

```diff
@@ -1,10 +1,115 @@
-{% from "./macros/common-components.njk" import goBack %}
+{% from "macros/common-components.njk" import goBack %}
```

Comment:
When you first naviagate to the page, the case name column indicates that the data is sorted in ascending order, however on first load no sort is applied.

Clicking the sort arrow fixes this, however I think this should also be sorted on the initial load

Reply:
I think sorting logic should be done on the BE, and we use mocked data. I will just change aria-sort to none for now? And once someone clicks on sort it should work as expected

Reply:
Sounds good :)